src/main-oe/sgxlkl_run_oe.c

```diff
@@ -1863,6 +1891,13 @@ int main(int argc, char* argv[], char* envp[])
     }
     sgxlkl_host_verbose_raw("result=%s\n", libsgxlkl);
 
+    sgxlkl_host_verbose("get_libsgxlkl_user_path... ");
+    if (!isolated_image_provided)
```

Comment:
This will always evaluate to true, I don't see where `isolated_image_provided` is being set again after initialization.

Reply:
Maybe should happen on L1774?